src/lib/services-manager/services_manager_profile.rb

```diff
@@ -128,6 +131,11 @@ def extract_target
       end
     end
 
+    # Filter out all YaST services
+    def reject_yast_services
+       self.services.reject! {|service| YAST_SERVICES.include?(service.name)}
```

Comment:
NP: according to our style guide, when using the `{}` block form, you should use 1-letter variable names. For example `{ |s| YAST_SERVICES.include?(s.name) }`.